### PR TITLE
add heros app

### PR DESCRIPTION
**Pull request by zubair-arbi**
@ayub-khan
Add `heros` app with basic models along with django admin.

### CODE REVIEW

File: heros/models.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    photo = models.ImageField(help_text='Hero picture.')
+    category = models.CharField(
+        max_length=25,
+        blank=False,
+        choices=HERO_CATEGORY_CHOICES,
+        help_text=_(
+            'Specify hero category from choices, i.e Strength.'
+        )
+    )
+    battle_style = models.CharField(
```
Comment (Ayub-Khan) [inline heros/models.py]: Change it to attack type here and in choice constant name

File: heros/models.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        help_text=_(
+            'Specify hero battle style from choices, i.e Malee.'
+        )
+    )
+
+    def __unicode__(self):
+        return _(u'{0}-{1}').format(self.name)
+
+
+class HeroType(models.Model):
```
Comment (Ayub-Khan) [inline heros/models.py]: Hero Role will be a batter choice i think

File: heros/models.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        ('initiator', 'Initiator'),
+    )
+    TYPE_LEVEL_CHOICES = (
+        (1, 'Basic type level'),
+        (2, 'Average type level'),
+        (3, 'Super type level'),
+    )
+
+    hero = models.ForeignKey(Hero, related_name='hero_types')
+    name = models.CharField(
```
Comment (Ayub-Khan) [inline heros/models.py]: change it to role

File: heros/models.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    )
+    level = models.CharField(
+        max_length=25,
+        blank=False,
+        choices=TYPE_LEVEL_CHOICES,
+        help_text=_(
+            'Specify level for the hero type, i.e Avergage Malee.'
+        )
+    )
+
```
Comment (Ayub-Khan) [inline heros/models.py]: another intersting thing add winrate. float

File: heros/models.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        ('complexity', 'Complexity'),
+        ('carry', 'Carry'),
+        ('support', 'Support'),
+        ('nuker', 'Nuker'),
+        ('disabler', 'Disabler'),
+        ('jungler', 'Jungler'),
+        ('durable', 'Durable'),
+        ('escape', 'Escape'),
+        ('pusher', 'Pusher'),
+        ('initiator', 'Initiator'),
```
Comment (Ayub-Khan) [inline heros/models.py:61]: add Roamer too

**Review by Ayub-Khan — CHANGES_REQUESTED** [review]

File: heros/models.py (hunk 4 above)
Comment (zubair-arbi) [reply heros/models.py]: win rate for a role?

File: heros/models.py (hunk 5 above)
Comment (zubair-arbi) [reply heros/models.py:61]: done

**Comment by zubair-arbi** [comment]
@Ayub-Khan please have another look.

**Review by Ayub-Khan — APPROVED** [review]

**Comment by Ayub-Khan** [comment]
Looks good.

**Pull request merged**